src/components/notifications/Cellsynt.vue

```diff
@@ -46,9 +48,12 @@ export default {
         HiddenInput
     },
     mounted() {
-        this.$parent.notification.cellsyntOriginator = "uptime-kuma";
-        this.$parent.notification.cellsyntOriginatortype = "alpha";
-        console.log("hello world");
+        if (typeof this.$parent.notification.cellsyntOriginatortype === "undefined" || this.$parent.notification.cellsyntOriginatortype === "") {
+            this.$parent.notification.cellsyntOriginatortype = "alpha";
+        }
+        if (typeof this.$parent.notification.cellsyntOriginator === "undefined" || this.$parent.notification.cellsyntOriginator === "") {
+            this.$parent.notification.cellsyntOriginator = "uptimekuma";
+        }
```

Comment:
the change you did here seems a bit overcompicated.

```suggestion
        this.$parent.notification.cellsyntOriginatortype ||= "alpha";
        this.$parent.notification.cellsyntOriginator ||= "uptimekuma";
```